ideascube/serveradmin/catalog.py

```diff
@@ -604,9 +606,20 @@ def list_nothandled(self, ids):
 
         return sorted(pkgs, key=attrgetter('id'))
 
+    @staticmethod
+    def _update_displayed_packages_on_home(*, to_remove_ids=None, to_add_ids=None):
+        displayed_packages = get_config('home-page', 'displayed-package-ids')
+        if to_remove_ids:
+            displayed_packages = [id for id in displayed_packages if id not in to_remove_ids]
+        if to_add_ids:
+            displayed_packages.extend(id for id in to_add_ids if id not in displayed_packages)
```

Comment:
Yes, but I want to preserve the order.
Not that it is really important cause the user cannot change this order for now.
But I would be surprising if cards' order changes when a package in installed or removed.